src/library/librarycontrol.h

```diff
@@ -1,7 +1,7 @@
 #ifndef LIBRARYMIDICONTROL_H
 #define LIBRARYMIDICONTROL_H
 
-#include <memory>
+#include <util/memory.h>
```

Comment:
since it's a mixxx local include and not a system include, could you use quotes instead of angle brackets? also move it down to the other local includes